README.md

```diff
@@ -1,69 +1,62 @@
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+## 📑 Table of Contents
+- [📊 WebMatrics ](#-WebMatrics-)
+  - [🛠 Built With ](#-built-with-)
+    - [🧰 Tech Stack ](#-tech-stack-)
+    - [💎 Key Features ](#-key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [📧 Contact ](#-contact-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [💖 Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Math Magicians <a name="about-project"></a>
+# 📊 WebMatrics <a name="about-project"></a>
 
-**Math Magicians** is a Single Page Application developed using React and Redux. We use the `create-react-app` build tool for development. The core component of the project is a calculator component that allows users to perform arithmetic operations.
+A web-based application to understand about the basics of React - Redux 
 
-## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+## 🛠 Built With <a name="built-with"></a>
+
+### 🧰 Tech Stack <a name="tech-stack"></a>
 
+- ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
+- ![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
+- ![React Router](https://img.shields.io/badge/React_Router-CA4245?style=for-the-badge&logo=react-router&logoColor=white)
+- ![Render](https://img.shields.io/badge/Render-%46E3B7.svg?style=for-the-badge&logo=render&logoColor=white)
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
+### 💎 Key Features <a name="key-features"></a>
 
-- **Interactive Calculator**
-- **Math quotes**
+- Generate a comprehensive Folder Structure for React App 💻
+- Understand the concept of setting up a project with redux 📈
+- Beautifully designed and easy-to-use 🎨
+- Continuous Monitoring 📸
+- Development Tools : Auto refresh and Hot Reload 🔗
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Link will be added after deployment
-
-<!-- - [Live Demo Link](https://yourdeployedapplicationlink.com) -->
+- See it live by clicking [Live Demo Link](https://web-metrics.netlify.app/)
```

Comment:
- [ ] ##### **Live version**

- [ ] As requested by the previous reviewer, It's one of the project requirements you deploy the application using Heroku, Netlify, or GitHub Pages. and make sure that  The project is deployed properly, and that the home and detail page works as expected. ✔️  Actually your deploy link does not display information on the details Page. Please fix this 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/227736261-6db326bc-8636-4460-ab38-85b979e3232c.png)


![image](https://user-images.githubusercontent.com/79058364/227736324-863351f5-b99c-4a0f-9742-bce04735cd95.png)